Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[dev-packages]
+
+
+
+[packages]
+
+"flake8" = "==3.5.0"
+"flake8-isort" = "==2.3"
+funcsigs = "==1.0.2"
+mozlog = "==3.7"
+PyPOM = "==1.2.0"
+pytest = "==3.3.2"
+pytest-metadata = "==1.5.1"
+pytest-selenium = "==1.11.4"
+pytest-variables = "==1.7.1"
+pytest-xdist = "==1.22.0"
+"querystringsafe_base64" = "==1.0.0"
+selenium = "==3.8.1"
```

Comment:
Add the following section to indicate that Python 2.7 is currently a requirement:

```ini
[requires]
python_version = "2.7"
```